.github/ISSUE_TEMPLATE/epic.yml

```diff
@@ -23,3 +23,10 @@ body:
     attributes:
       label: Additional context
       description: Add any other context or screenshots here.
+
+  - type: textarea
```

Comment:
Most epics created lately are QE epics where this wouldn't apply.
The ones development is using are typically only for grouping, e.g. we need to do 4 features in order to support X. Those features are independent and could be delivered over 3 sprints (or never finished) so IMHO it would be better to align QE work directly to the features than to the epic.

Reply:
Make sense. thx.